include/deal.II/grid/reference_cell.h

```diff
@@ -834,19 +834,26 @@ namespace ReferenceCell
           const unsigned int  face,
           const unsigned char face_orientation) const override
         {
-          Assert(false, ExcNotImplemented());
-
-          (void)face;
-
-          static const std::array<std::array<unsigned int, 3>, 6> table = {
-            {{{2, 1, 0}},
-             {{0, 1, 2}},
-             {{1, 2, 0}},
-             {{0, 2, 1}},
-             {{1, 0, 2}},
-             {{2, 0, 1}}}};
+          if (face == 1) // QUAD
```

Comment:
Isn't face 0 the quadrilateral? That's what #11095 depicts.

Reply:
You are right. I have fixed it!